src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx

```diff
@@ -169,7 +170,8 @@ export class PopoverOverlay extends PureComponent<PopoverOverlayProps, State> {
         return;
       }
 
-      this.contentNode.current.focus({
+      const focusableChild = findFirstFocusableNode(this.contentNode.current);
```

Comment:
If you look at all existing storybook stories you will see that `findFirstFocusableNode` will always find a node and focus it. 
<img width="918" alt="Screen Shot 2020-11-04 at 2 03 02 PM" src="https://user-images.githubusercontent.com/5626729/98157028-b7ae7400-1ea6-11eb-8bb8-445ef16eab82.png">

I don't think that's an issue but are okay to change the default behaviour of the component?

Reply:
I think we should make it an optional feature as it can look weird in some cases to have a focus ring around a button for example.

Reply:
`preventAutofocus` could be deprecated and replaced by an enum prop? 
E.g. it could be `defaultFocus`, by default it would be `Container`, we could set `FirstNode` or `None`.

Reply:
Sounds good, although I am not familiar with the deprecation process 🙂 
I would propose another name for it, maybe `autoFocusTarget`. What do you think?

Reply:
Sounds good, we can try to contact polaris tomorrow to see if that's doable.

Reply:
Here are the deprecation guidelines: https://github.com/Shopify/polaris-react/blob/master/documentation/Deprecation%20guidelines.md